marklogic-data-hub-central/ui/e2e/cypress/integration/modeling/graphValidations.spec.tsx

```diff
@@ -249,11 +249,11 @@ describe("Graph Validations", () => {
     cy.publishDataModel();
 
     cy.log("**Deleting relation, entities and publish**");
-    propertyModal.getDeleteIcon("Test1-relTest1Test2").click();
+    propertyModal.getDeleteIcon("Test1-relTest1Test2").should("be.visible").scrollIntoView().click();
```

Comment:
we should put the scroll into view before checking for its visibility. Please let's make that change for all the following with the same